policy-report/samples/sample-rhacm-policy.yaml

```diff
@@ -16,16 +15,10 @@ summary:
 results:
 - policy: mustnothaveimagevuln
   message: must not have imagemanifestvulns
-  status: Fail
+  result: fail
   scored: false
   resources:
     - apiVersion: secscan.quay.redhat.com/v1alpha1
       kind: ImageManifestVuln
       name: sha256.8d104847fc2371a983f7cb01c7c0a3ab35b7381d6bf7ce355d9b32a08c0031f0
       namespace: openshift-cluster-version
-  data:
```

Comment:
@mritunjaysharma394 - please add back `data` as it is a supported field.

Reply:
While adding `data`, I was getting this error: 

```
error: error validating "samples/sample-rhacm-policy.yaml": error validating data: ValidationError(PolicyReport.results[0]): unknown field "data" in io.wgpolicyk8s.v1alpha1.PolicyReport.results; if you choose to ignore these errors, turn validation off with --validate=false

```

On checking this again (https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/api/v1alpha1/policyreport_types.go), I realised that `data`  might have been renamed as `properties`. Should I add `data` as `properties`?